[feat] new engine: wikispecies

Bnyro opened the pull request:
## What does this PR do?
* adds wikispecies as an engine
* wikispecies is just a standard wikimedia api

## Why is this change important?
* it contains articles about different animals/species and other forms of life.

## How to test this PR locally?
* disable all engines except and enable wikispecies

## Related issues

closes  #1222

Bnyro committed:
[feat] new engine: wikispecies

return42 commented on searx/engines/mediawiki.py:
```diff
@@ -86,7 +86,7 @@ def response(resp):
 
         # append result
-        results.append({'url': url, 'title': result['title'], 'content': ''})
+        results.append({'url': url, 'title': result['title'], 'content': result.get('snippet', '')})
```
Wow this is *rocket cool*, it (BTW) fixes the content field of wikibooks, wikinews, wikiquote .. and all the other mediawiki engines :+1:

In my review I will strip the HTML tags from the *snippet* .. coming soon.

return42 committed:
[mod] engine: wikimedia - improve results, add addition settings & doc

Signed-off-by: Markus Heiser <markus.heiser@darmarit.de>

return42 committed:
[mod] settings.yml: enable engines from Wikimedia Foundation

The Wikimedia wikis [1] engines provide good answers and have short response
times --> no reason to disable these enhgines by default.  BTW: this patch adds
a (sub-) category ``wikimedia`` for the engines [1].

[1] https://meta.wikimedia.org/wiki/Wikimedia_wikis

Signed-off-by: Markus Heiser <markus.heiser@darmarit.de>
